src/utils/profitAndLossUtils.ts

```diff
@@ -4,13 +4,11 @@ import { ProfitAndLoss } from '../types/profit_and_loss'
 
 const doesLineItemQualifies = (item: LineItem) => {
   return !(
-    item.is_contra ||
-    item.value === undefined ||
-    item.value === null ||
-    isNaN(item.value) ||
-    item.value === -Infinity ||
-    item.value === Infinity ||
-    item.value < 0
+    item.value === undefined
```

Comment:
we were filtering out everything here -> now we're pushing that responsibility to the very end when we format the chart